_write.js

```diff
@@ -57,6 +51,16 @@ module.exports = function _write(httpMethod, options, callback) {
       catch (e) {
         err = e
       }
+
+      if (!ok) {
+        err = Error(httpMethod + ' failed with: ' + res.statusCode)
+        err.raw = res
```

Comment:
lets not add this `err.raw = res` (since we are adding result already)

Reply:
Fundamentally different things though right? Without the raw response we can't get at the headers and status code from the underlying response.

Reply:
Could just add `err.status` and `err.headers` if that was preferable though.